kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -2,10 +2,10 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: org.eclipse.kura.api
 Bundle-SymbolicName: org.eclipse.kura.api;singleton:=true
-Bundle-Version: 2.4.0.qualifier
```

Comment:
There should be no need for a major version bump

Reply:
baseline check plugin said me it is required.